powershell-adapter/Tests/winps_resource.dsc.yaml

```diff
@@ -1,11 +1,14 @@
-# Example configuration mixing native app resources with classic PS resources
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
 $schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/10/config/document.json
 resources:
 - name: Get info from classic DSC resources
-  type: Microsoft.Windows/WindowsPowerShell
+  type: Microsoft.DSC/WindowsPowerShell
```

Comment:
Since WindowsPowerShell is part of Windows, we want the typename of this resource to be `Microsoft.Windows/WindowsPowerShell` whereas PowerShell 7 being cross platform would be `Microsoft.DSC/PowerShell`

Reply:
fixed